core/templates/msg.html

```diff
@@ -17,64 +16,113 @@
 </div>
 <%});
 %>
-
 </div>
+</div>
+</section>
+<section id="right_panel">
+<p id="tasks">Tasks</p>
+<button id="add_new" onClick="newTask()">Add New</button>
+<div id="add_task"></div>
+<form id="task_sort">
+<% all_tasks.rows.forEach(function(rows){%>
+<div class="projects" data-number='<%=rows.task_id%>' id='<%=rows.task_id%>' ><%=rows.task %></div>
+<%});
+%>
+</form>
+
 <script>
```

Comment:
How hard would it be to move this script to a separate file? It would help to declutter the HTML